src/github/interface.ts

```diff
@@ -56,6 +56,7 @@ export interface IGitHubRef {
 
 export interface ILabel {
 	name: string;
+	color: string;

```

Comment:
I think that colour shouldn't be a required field though. Colour should be a color?:string interface.

That way if someone has no interest in colouring the labels, they can keep on using the current implementation.